include/fc/log/zipkin.hpp

```diff
@@ -45,6 +45,10 @@ class zipkin_config {
    /// Starts with a random id and increments on each call, will not return 0
    static uint64_t get_next_unique_id();
 
+   /// Signal safe
+   /// This is not called directly from the SIGHUP signal handler in the main thread
```

Comment:
This is only true for your use case. Other use cases might use something other than the main thread.

Reply:
Updated the comments and Change Description of the PR.